.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+---
+# desc: lint sanity check
+name: lint
+
+on:
+  push:
+  workflow_dispatch:
+
+
+jobs:
+  lint:
+    name: pre-commit
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
+
+      - uses: terraform-linters/setup-tflint@19a52fbac37dacb22a09518e4ef6ee234f2d4987 # v4
+        name: Setup TFLint
+        with:
+          tflint_version: v0.50.3
+
+      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5
+      - run: go install github.com/terraform-docs/terraform-docs@v0.17.0
```

Comment:
maybe makes sense to switch to using asdf for that?
https://github.com/asdf-vm/actions?tab=readme-ov-file#setup

Generally Python is available on the runner, not sure why to specifically add it but nevertheless, could be added to `.tool-versions` as well.

Results in you not having to maintain the versions in different files.

Reply:
or as mentioned in another comment, pull the versions from `.tool-versions` https://github.com/endorama/asdf-parse-tool-versions

Reply:
It's still the same.

I'd suggest pulling the version from `.tool-versions` and then installing like

```
    - name: Install Terraform Docs
      uses: jaxxstorm/action-install-gh-release@v1.10.0
      with:
        repo: terraform-docs/terraform-docs
        tag: v0.17.0
```